libs/ballot-interpreter/src/interpret_vx_hmpb.test.ts

```diff
@@ -219,6 +219,38 @@ describe('HMPB - Famous Names', () => {
       blankPageInterpretation
     );
   });
+
+  test('streaks on ballot', async () => {
+    const images = asSheet(await pdfToPageImages(blankBallotPath).toArray());
+    const [frontImage, backImage] = images;
+    const canvas = createCanvas(frontImage.width, frontImage.height);
+    const context = canvas.getContext('2d');
+    context.imageSmoothingEnabled = false;
+    context.putImageData(frontImage, 0, 0);
+    context.fillStyle = 'black';
+    context.fillRect(canvas.width / 2, 0, 1, canvas.height);
+    const streakImage = context.getImageData(0, 0, canvas.width, canvas.height);
```

Comment:
This being a test the performance doesn't matter _that_ much, but if you want to directly edit the image data this is how I'd do it:

```suggestion
    const bytesPerRow = frontImage.width * 4;
    for (
      let offset = bytesPerRow / 2;
      offset < frontImage.data.length;
      offset += bytesPerRow
    ) {
      frontImage.data[offset] = 0;
      frontImage.data[offset + 1] = 0;
      frontImage.data[offset + 2] = 0;
      frontImage.data[offset + 3] = 255;
    }
```

Reply:
Seems more complicated to me since you have to understand the image data format